### ISSUE TITLE
Notifications – Re-enable in corresponding `netlify.toml` files used in deployment workflow

### ISSUE THREAD

**Issue by kepae** [responsible-ai-collaborative/aiid#2622]
I believe we are currently a week behind on notifications because we are not using the correct configuration for the `netlify build` step of our deployment.

The `deploy-netlify.toml` file is used in the `deploy.yml` workflow file:
https://github.com/responsible-ai-collaborative/aiid/blob/ecd60574d74f05c4a0277b966d27ab1677e91609/.github/workflows/deploy.yml#L58

Which is called by the production deploy workflow:
https://github.com/responsible-ai-collaborative/aiid/actions/runs/7819132023/workflow#L37

However, the plug-in for processing notifications is disabled here:
`site/gatsby-site/deploy-netlify.toml`
https://github.com/responsible-ai-collaborative/aiid/blob/ecd60574d74f05c4a0277b966d27ab1677e91609/site/gatsby-site/deploy-netlify.toml#L12

rather, it's enabled in the old `netlify.toml` file:
`site/gatsby-site/netlify.toml`
https://github.com/responsible-ai-collaborative/aiid/blob/ecd60574d74f05c4a0277b966d27ab1677e91609/site/gatsby-site/netlify.toml#L23

**Comment by kepae** [comment]
I think after #2625 we should retire this file unless we want to keep it for pushing to Netlify for some reason.
https://github.com/responsible-ai-collaborative/aiid/blob/ecd60574d74f05c4a0277b966d27ab1677e91609/site/gatsby-site/netlify.toml#L23

In which case, we need to document the heck out of these file dependencies for the workflows.

**Comment by pdcp1** [comment]
@cesarvarela Do you have a draft of the workflows documentation? We should replace the old `netlify.toml`

**Comment by cesarvarela** [comment]
What I had was already merged 🤔

**Comment by pdcp1** [comment]
Great! I will replace/delete the old `netlify.toml` to avoid confusion

**Comment by pdcp1** [comment]
This ticket is now being closed as it has already been merged into Staging.
